### PR TITLE
[StreamRecorder] Handle NotSupportedException as new feature added

### PR DESCRIPTION
**Pull request by hancepark**
-Tizen.Multimedia.StreamRecorder.StreamRecorder()

Signed-off-by: Hyunsoo Park <hance.park@samsung.com>

### Description of Change ###

feature is added. (http://tizen.org/feature/multimedia.streamrecorder.record)
So related APIs are modified.

### API Changes ###

 - ACR:TCSACR-305
(https://code.sec.samsung.net/jira/browse/TCSACR-305)

### CODE REVIEW

**Comment by hsgwon** [comment]
Please add `[StreamRecorder]` prefix to the title.

File: src/Tizen.Multimedia.StreamRecorder/StreamRecorder/StreamRecorder.cs [hunk 1]
```diff
@@ -34,6 +34,12 @@ public partial class StreamRecorder : IDisposable
         private bool _audioEnabled;
         private bool _videoEnabled;
         private StreamRecorderVideoFormat _sourceFormat;
+        private const string Feature = "http://tizen.org/feature/network.streamrecorder.record";
+
+        private static bool IsSupported()
+        {
+            return System.Information.TryGetValue(Feature, out bool isSupported) ? isSupported : false;
```
Comment (hsgwon) [inline src/Tizen.Multimedia.StreamRecorder/StreamRecorder/StreamRecorder.cs]: You can do the same thing just as below code.

```suggestion
            return System.Information.TryGetValue(Feature, out bool isSupported) && isSupported;
```

**Comment by hsgwon** [comment]
Please add `[StreamRecorder]` prefix to the tile.

**Comment by hancepark** [comment]
> Please add `[StreamRecorder]` prefix to the tile.

done. Thank you

File: src/Tizen.Multimedia.StreamRecorder/StreamRecorder/StreamRecorder.cs (hunk 1 above)
Comment (hancepark) [reply src/Tizen.Multimedia.StreamRecorder/StreamRecorder/StreamRecorder.cs]: Thank you, i will take

**Comment by WonyoungChoi** [comment]
@hancepark Please make a PR for API8 also.

File: src/Tizen.Multimedia.StreamRecorder/StreamRecorder/StreamRecorder.cs (hunk 1 above)
Comment (idkiller) [reply src/Tizen.Multimedia.StreamRecorder/StreamRecorder/StreamRecorder.cs]: hskwon is right.

truth table can reveal it.

ternary operator way.
| return \ isSupported | true | false |
| --- | --- | --- |
|  true |  true | false |
| false | false | false |

&& way
| return \ isSupported | true | false |
| --- | --- | --- |
|  true |  true | false |
| false | false | false |

Comment (hsgwon) [reply src/Tizen.Multimedia.StreamRecorder/StreamRecorder/StreamRecorder.cs]: > It is out bool, Your suggestion always TRUE in my opinion

I didn't check your comment early, and I don't understand why you thought it's always true.

**Comment by hancepark** [comment]
> @hancepark Please make a PR for API8 also.

**Comment by hancepark** [comment]
@WonyoungChoi  Thank you, I will make PR for API8

**Comment by hancepark** [comment]
ACR is accepted. Please merge it.

**Pull request merged**